stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -36,506 +39,8 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
-  // C standard library
-  module C {
-% if CMAKE_SDK in ["LINUX", "CYGWIN"]:
-    module features {
-% if CMAKE_SDK == "LINUX":
-      header "${GLIBC_INCLUDE_PATH}/stdc-predef.h"
-% end
-      header "${GLIBC_INCLUDE_PATH}/features.h"
-      export *
-    }
-% end
-% if CMAKE_SDK in ["LINUX", "FREEBSD", "CYGWIN", "HAIKU"]:
-    module complex {
-      header "${GLIBC_INCLUDE_PATH}/complex.h"
-      export *
-    }
-% end
-% if CMAKE_SDK in ["LINUX", "CYGWIN"]:
-    module pty {
-      header "${GLIBC_INCLUDE_PATH}/pty.h"
-      export *
-    }
-    module utmp {
-      header "${GLIBC_INCLUDE_PATH}/utmp.h"
-      export *
-    }
-% end
-% if CMAKE_SDK == "FREEBSD":
-    module pty {
-      header "${GLIBC_INCLUDE_PATH}/libutil.h"
-      export *
-    }
-    module utmp {
-      header "${GLIBC_INCLUDE_PATH}/utmpx.h"
-      export *
-    }
-% end
-% if CMAKE_SDK == "HAIKU":
-    module pty {
-      header "${GLIBC_INCLUDE_PATH}/../bsd/pty.h"
-      export *
-    }
-% end
-
-    module ctype {
-      header "${GLIBC_INCLUDE_PATH}/ctype.h"
-      export *
-    }
-    module errno {
-      header "${GLIBC_INCLUDE_PATH}/errno.h"
-      export *
-    }
-
-    module fenv {
-      header "${GLIBC_INCLUDE_PATH}/fenv.h"
-      export *
-    }
-
-    // note: supplied by compiler
-    // module float {
-    //   header "${GLIBC_INCLUDE_PATH}/float.h"
-    //   export *
-    // }
-
-    module inttypes {
-      header "${GLIBC_INCLUDE_PATH}/inttypes.h"
-      export *
-    }
-
-    // note: potentially supplied by compiler
-    // module iso646 {
-    //   header "${GLIBC_INCLUDE_PATH}/iso646.h"
-    //   export *
-    // }
-    // module limits {
-    //   header "${GLIBC_INCLUDE_PATH}/limits.h"
-    //   export *
-    // }
-
-    module locale {
-      header "${GLIBC_INCLUDE_PATH}/locale.h"
-      export *
-    }
-    module math {
-% if CMAKE_SDK == "LINUX":
-      link "m"
-% end
-      header "${GLIBC_INCLUDE_PATH}/math.h"
-      export *
-    }
-% if CMAKE_SDK != "WASI":
```

Comment:
I assume we're going to need to add these `WASI` checks in `SwiftGlibc.h.gyb`.

Reply:
AFAIK there's no need to, since we're now using `__has_include` guard in the header. What we had to carefully exclude from the file before at file-generation time is now handled by this at include time.